generator/dart/packages/generated/google_cloud_protobuf/.sidekick.toml

```diff
@@ -0,0 +1,26 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+[general]
+specification-source = 'google/protobuf'
+
+[source]
+name-override = 'protobuf'
+title-override = 'Core Protobuf Types'
+description-override = 'Core Protobuf types used by most services.'
+include-list = "any.proto,api.proto,field_mask.proto,source_context.proto,type.proto"
```

Comment:
I would prefer more incremental changes.  Maybe just start with `field_mask.proto` and add the other files one at a time?

Reply:
Sounds good - will drop down to just including field_mask here.